torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -225,6 +240,13 @@ def _share_state_and_init_handle_attrs(
         )
         if handle._has_optim_in_backward:
             torch._C._log_api_usage_once("fsdp.optimizer_in_backward")
+    # Keep track of any new unshard streams we may have to add for specific process groups.
+    fsdp_pg_unshard_streams = {}
+    try:
+        unshard_priority = root_state._unshard_stream.priority
```

Comment:
Can we just add a priority attribute to the Stream class and initialize it to 0 or something??

Reply:
resolved by change below